packages/astro/src/core/build/plugins/plugin-internals.ts

```diff
@@ -41,6 +41,7 @@ export function vitePluginInternals(input: Set<string>, internals: BuildInternal
 			}
 			await Promise.all(promises);
 			for (const [, chunk] of Object.entries(bundle)) {
+				// why is the distinction important?
 				if (chunk.type === 'chunk' && chunk.facadeModuleId) {
```

Comment:
<details>
<summary>how it's used in the codebase</summary>

For distinguishing which chunks get added to `internals.pageToBundleMap` and which to `internals.entrySpecifierToBundleMap`. Only the chunks in `pageToBundleMap` are considered when turning prerendered pages into no-op modules.

</details>

`facadeModuleId` seems to be a barely documented property added by rollup. Sometimes, whether or not an astro page gets it depends on whether an unrelated page was prerendered. The end result is, we are not reliably turning prerendered components into no-ops.

Reply:
I think this is checking of the chunk is an entrypoint. Usually a page, I believe.